Client/Frontend/Browser/TopTabsViewController.swift

```diff
@@ -254,11 +254,11 @@ extension TopTabsViewController: UICollectionViewDataSource {
         tabCell.titleText.text = tab.displayTitle
         
         if tab.displayTitle.isEmpty {
-            if (tab.url?.baseDomain()?.contains("localhost") ?? true) {
+            if (tab.webView?.URL?.baseDomain()?.contains("localhost") ?? true) {
                 tabCell.titleText.text = AppMenuConfiguration.NewTabTitleString
             }
             else {
-                tabCell.titleText.text = tab.displayURL?.absoluteString
+                tabCell.titleText.text = tab.webView?.URL?.absoluteDisplayString()
```

Comment:
Yeah, sorry, I should have said why I changed that. The displayURL is nil until the tab is fully loaded, so it will show "New Tab" for a few seconds until the webpage is loaded. But the webView.URL is available immediately so it will show the URL in the tab title until it loads then it's replaced with the title of the webpage.